tests/semantic_analyser.cpp

```diff
@@ -30,9 +30,13 @@ void test_for_warning(BPFtrace &bpftrace,
   clang.parse(driver.ctx.root, bpftrace);
 
   ASSERT_EQ(driver.parse_str(input), 0);
-  std::stringstream out;
-  ast::SemanticAnalyser semantics(driver.ctx, bpftrace, out);
+  // Override to mockbpffeature.
```

Comment:
unnecessary comment

Reply:
Hm, I think this actually got copied up from this comment in the other `test()` in this file. Removed. Not sure what it means in the other case :)